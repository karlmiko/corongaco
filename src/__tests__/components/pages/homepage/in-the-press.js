import React from 'react'
import renderer from 'react-test-renderer'
import { useStaticQuery } from 'gatsby'
import InThePress from '~components/pages/homepage/in-the-press'

beforeEach(() => {
  useStaticQuery.mockImplementation(() => ({
    allCovidPress: {
      edges: [
        {
          node: {
            id: '5ce2bc4d-3d24-5e1a-91bb-a30d098ccfc7',
            title:
              'Coronavirus testin: California continues to lag compared to other states',
            url:
              'https://abc7news.com/coronavirus-testing-california-covid-19-in-tests-ca-pandemic/6111601/',
            publication: 'ABC 7 News',
            publishDate: 'April 18, 2020',
          },
        },
        {
          node: {
            id: '68c6166f-7498-5eaf-bd6a-178a5e13acf2',
            title:
              'Amid flurry of national coronavirus data, Massachusetts remains a hot spot of infection',
            url:
              'https://www.bostonglobe.com/2020/04/18/nation/amid-flurry-national-coronavirus-data-massachusetts-remains-hot-spot-infection/',
            publication: 'The Boston Globe',
            publishDate: 'April 18, 2020',
          },
        },
      ],
    },
  }))
})

describe('Components : Pages : Homepage : In the Press', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<InThePress />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
