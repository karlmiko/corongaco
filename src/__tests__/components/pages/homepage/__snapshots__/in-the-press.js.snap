// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : Homepage : In the Press renders correctly 1`] = `
<ul
  className="press-list pressList"
>
  <li>
    <cite
      className="title"
    >
      <a
        href="https://abc7news.com/coronavirus-testing-california-covid-19-in-tests-ca-pandemic/6111601/"
      >
        Coronavirus testin: California continues to lag compared to other states
      </a>
    </cite>
    <div
      className="detail-text detailText"
    >
      <span
        className="source"
      >
        ABC 7 News
      </span>
      <span
        aria-hidden={true}
        className="dotSeparator"
      >
        •
      </span>
      April 18, 2020
    </div>
  </li>
  <li>
    <cite
      className="title"
    >
      <a
        href="https://www.bostonglobe.com/2020/04/18/nation/amid-flurry-national-coronavirus-data-massachusetts-remains-hot-spot-infection/"
      >
        Amid flurry of national coronavirus data, Massachusetts remains a hot spot of infection
      </a>
    </cite>
    <div
      className="detail-text detailText"
    >
      <span
        className="source"
      >
        The Boston Globe
      </span>
      <span
        aria-hidden={true}
        className="dotSeparator"
      >
        •
      </span>
      April 18, 2020
    </div>
  </li>
</ul>
`;
